### PR TITLE
Expanding the books page to also include videos.

### PR DESCRIPTION
**Pull request by Sfshaza**
And adding a video published by Packt.

Fixed: https://github.com/dart-lang/site-www/issues/134

Staged: https://sz-www-2.firebaseapp.com/resources/books

@kwalrath, please review

### CODE REVIEW

File: src/resources/books.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 layout: default
 permalink: /resources/books
-title: Books about Dart
+title: Books and Videos about Dart
 description: Read all about it! Here's a collection of books about Dart.
 toc: false
 ---
 
-Here's a collection of books about Dart.
+Here's a collection of books and videos about Dart.
```
Comment (kwalrath) [inline src/resources/books.md]: Should we qualify the "videos" part somehow? We're not including our videos (although maybe we should point to them from here). And I assume we won't include short videos. We're really talking about major resources here. I'm not sure what words to use, though...

File: src/resources/books.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 [Let us know](https://github.com/dart-lang/site-www/issues)
-if you find another book that we should add.
+if you find another resource that we should add.
```
Comment (kwalrath) [inline src/resources/books.md]: My recent dive into the style guide has informed me that we should almost always put the qualifier before the "do this" thing. So:

If you ...., [let us know](...).

File: src/resources/books.md (hunk 1 above)
Comment (Sfshaza) [inline src/resources/books.md]: I thought about this, too. We don't want to include  youtube videos or Dart Tips, for example. It seems like we shouldn't have to explain it but I could, perhaps, add "packaged" videos. Not loving it, though. I could say, "professionally produced books and videos", but I'm not loving that, either. I think we should leave it be and just vet any thing that comes in.

**Comment by Sfshaza** [comment]
Staged: https://sz-www-2.firebaseapp.com/resources/books

@kwalrath

File: src/_tutorials/dart-vm/cmdline.md [hunk 3]
```diff
@@ -29,7 +29,11 @@ prevpage:
 <aside class="alert alert-info" markdown="1">
 <strong>Prerequisite:</strong>
 
+<<<<<<< Updated upstream
```
Comment (kwalrath) [inline src/_tutorials/dart-vm/cmdline.md]: Uh oh, fix this!

**Comment by kwalrath** [comment]
LGTM!

**Pull request merged**